digitalforms-api/pom.xml

```diff
@@ -69,6 +69,30 @@
 			</exclusions>
 		</dependency>
 
+		<!-- Junit 5 -->
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-api</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter-engine</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.junit.platform</groupId>
+			<artifactId>junit-platform-launcher</artifactId>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Isn't this already included in spring-boot-starter-test dependency?
https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter-test/2.2.7.RELEASE

Reply:
Yup. Checked. Good catch. No need to have this in the POM. Thanks!